internal/engine/wazevo/engine.go

```diff
@@ -841,3 +897,14 @@ func (cm *compiledModule) getSourceOffset(pc uintptr) uint64 {
 	}
 	return cm.sourceMap.wasmBinaryOffsets[index]
 }
+
+func sequence(size int) <-chan int {
+	result := make(chan int)
+	go func() {
+		for i := range size {
+			result <- i
+		}
+		close(result)
+	}()
+	return result
```

Comment:
could you remove this sequence function ? i think it's overkill and don't want to spawn channel/goroutines unnecessarily for the worker=1 case (which is the default for existing projects.)

Reply:
sounds good.